soda-consumer-scala/src/main/scala/com/socrata/soda2/values/SodaValue.scala

```diff
@@ -353,21 +357,22 @@ case class SodaMultiPoint(points: Seq[SodaPoint]) extends SodaValue {
 }
 
 case object SodaMultiPoint extends SodaType with (Seq[SodaPoint] => SodaMultiPoint) {
-  implicit val jsonCodec = new JsonCodec[SodaMultiPoint] {
-    val points = Variable[Seq[SodaPoint]]()(JsonCodec.seqCodec(SodaPoint.simplePoint, implicitly))
+  implicit val jsonCodec = new JsonEncode[SodaMultiPoint] with JsonDecode[SodaMultiPoint]
+  {
+    val points = Variable[Seq[SodaPoint]]
```

Comment:
All the places that were referring to `SodaPoint.simplePoint` still need to do so; it's a different encoding from the implicit `SodaPoint` format, which is a geojson point.

Reply:
fixed